docs/cases/indicators-of-compromise.asciidoc

```diff
@@ -143,4 +143,16 @@ NOTE: Some threat intelligence feeds provide  https://www.cisa.gov/tlp#:~:text=I
 To remove an indicator attached to a case, click *More actions* (*...​*) -> *Delete attachment* in the case comment. 
 
 [role="screenshot"]
-image::images/remove-indicator.png[Removing an indicator from a case]
\ No newline at end of file
+image::images/remove-indicator.png[Removing an indicator from a case]
+
+[discrete]
+[[add-indicator-to-blocklist]]
+== Use data from indicators to expand the blocklist
+
+Add indicator values to the <<blocklist,blocklist>> to prevent selected applications from running on your hosts. You can use MD5, SHA-1, or SHA-256 hash values from `file` type indicators. 
+
+You can add indicator values to the blocklist from the Indicators table or the Indicator details flyout. From the Indicators table, select *More actions* (*...​*) -> *Add blocklist entry*.  Alternatively, open an indicator's details, then select *Take action -> Add blocklist entry*. 
```

Comment:
`More actions` and `Take action` are technically menus according to UX, so we'd need to phrase them like `Select the <> menu, then <>`.  

Also, confirming we've capped `Indicator details flyout`- I know we lowercase `alert details flyout` so just checking.